Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -412,6 +414,31 @@ StringView string_from_property_id(PropertyID property_id) {
     }
 }
 
+bool is_animatable_property(PropertyID property_id)
+{
+    switch (property_id) {
+)~~~");
+
+    properties.for_each_member([&](auto& name, auto& value) {
+        VERIFY(value.is_object());
+        auto animation_type = value.as_object().get_bool("animatable"sv);
+        if (!animation_type.has_value() || !animation_type.value())
+            return;
```

Comment:
From [Web Animations 1](https://drafts.csswg.org/web-animations-1/#animating-properties):
> Unless otherwise specified, all CSS properties are animatable.

So, we could get away with just defaulting to "true" if no value is specified. This would save a couple hundred lines of JSON.

Then again, different properties animate in different ways - that spec link above lists them. It would probably be useful to put that information in the JSON instead. Something like `"animation-type": "not-animatable" | "discrete" | "by-computed-value" | "repeatable-list" | "other"`. I've no idea if/how we handle discrete animated properties currently.

Reply:
Yeah that's probably the better route in the long run, I don't use that information right now, but I assume it's supposed to be used in `StyleComputer::interpolate_property` somehow. Not quite sure what that would look like, but that's a problem for later

Reply:
Switched to an `"animation-type"` key. Also included a way to get the type from a property even though I don't need it in this PR.

Reply:
Thought about this some more (after learning a lot more about CSS internals and how the `StyleComputer` works), it is absolutely useful; a simple check at the top of `interpolate_property` for `CSS::animation_type_from_property(property_id) == CSS::AnimationType::Discrete` would allow us to handle discrete properties easily, and presumably `repeatable-list` would be similar (which seems obvious in hindsight). I'll definitely leave it in this PR.